DESCRIPTION

```diff
@@ -1,10 +1,11 @@
 Package: seeker
 Type: Package
 Title: Process Genomic Data
-Version: 1.0.2
+Version: 1.0.3
 Authors@R: c(
-  person('Jake', 'Hughey', , 'jakejhughey@gmail.com', c('aut', 'cre')),
-  person('Josh', 'Schoenbachler', , 'josh.schoenbachler@gmail.com', 'aut'))
+    person("Jake", "Hughey", , "josh.schoenbachler@gmail.com", role = c("aut", "cre")),
```

Comment:
Fix email address. Was it necessary to change indentation and quotation marks?

Reply:
I don't remember changing this, nor do I think I would have had any reason to. Is it possible that documenting the package or another step could have changed/updated it?

Reply:
Email address still not fixed. Please be careful and do not rush.